src/translator_adapter.h

```diff
@@ -38,6 +38,15 @@ class TranslatorAdapterBase : public Translator
     virtual QCString updateNeededMessage() override = 0;
 };
 
+class TranslatorAdapter_1_13_0 : public TranslatorAdapterBase
+{
+  public:
+    QCString updateNeededMessage() override
+    { return createUpdateNeededMessage(idLanguage(),"release 1.13.0"); }
+    QCString trImportant() override
+    { return english.trImportant(); }
+};
+
```

Comment:
Changes for translations are very welcome.
Changes for the translation are not related to  the pascal / dbase implementation and have to be handled in a separate PR.  (No PR hijacking)

Furthermore there is no need  for adding a 1_13_0 translator adapter, when updating to the current version (which is anyway 1.13.2)  in the translator file the base class `Translator` should be used.